online-shop/src/app/components/containers/products-details/products-details.component.spec.ts

```diff
@@ -0,0 +1,21 @@
+import { ComponentFixture, TestBed } from '@angular/core/testing';
```

Comment:
This file can be removed, since you haven't written any tests. This applies for all occurrences where you have spec files.